widgetssdk/src/main/java/com/glia/widgets/callvisualizer/ActivityWatcherForCallVisualizerController.kt

```diff
@@ -138,19 +142,13 @@ internal class ActivityWatcherForCallVisualizerController(
         }
     }
 
-    override fun onLink1Clicked() {
-        Logger.d(TAG, "onLink1Clicked")
-        watcher.openWebBrowserActivity(
-            stringProvider.getRemoteString(R.string.engagement_confirm_link1_text),
-            stringProvider.getRemoteString(R.string.engagement_confirm_link1_url)
-        )
-    }
+    override fun getConfirmationDialogLinks() = confirmationDialogLinksUseCase(stringProvider)
```

Comment:
I think that the usecase could already have the provider from creation

Reply:
Fixed 187f4b051dc8862adfc04b296a92b3d75c7891be